CMSgov/mint-app
# [EASI-2412] Read Only CR and TDLs Subpage

# EASI-2412

## Changes and Description

- Add `ReadOnlyCRTDLS` component to the Read Only page
- Add new fields to `GetModelSummary` query
- Pass down `readOnly` prop to the CRTDL Table to it adjusts how user will navigate back and forth in the `AddCRTDL` page via hash location `#read-only`

## AC

- CRTDL component on readonly
- Similar layout to discussions (hide contact right side bar)
- Edit access - Edit should take user to edit page for CRTDL


 ## How to test this change

1. Preseed data
2. Navigate to the `CRs and TDLs` section of the read only page for `Plan With CRs and TDLs`
3. Ensure Table shows the correct items
4. Ensure Edit access is only available for collabortaors and assesment peopleepwewe

Redo test not as a collaborator and not having assessment job code
1. Ensure that `Actions` column is MIA


## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
